manim/mobject/svg/text_mobject.py

```diff
@@ -339,122 +318,206 @@ def text2svg(self):
             if setting.line_num != last_line_num:
                 offset_x = 0
                 last_line_num = setting.line_num
-            context.move_to(START_X + offset_x, START_Y + lsh * setting.line_num)
+            context.move_to(
+                START_X + offset_x, START_Y + line_spacing * setting.line_num
+            )
             context.show_text(text)
             offset_x += context.text_extents(text)[4]
+        surface.finish()
+        return file_name
+
+
+# Following class is just a Little implementation of upcomming feautures. Ignore it for now.
```

Comment:
Should still have docstrings..

Reply:
I will maybe create some different `Mobject` only for backgrounds of other `Mobjects`